calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+| `radar_foreground_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Foreground pointcloud from radar.               |
+| `radar_detection_markers`     | `visualization_msgs::msg::MarkerArray` | Radar detections.                               |
+| `matches_markers`             | `visualization_msgs::msg::MarkerArray` | Matched lidar and radar detection.              |
+| `tracking_markers`            | `visualization_msgs::msg::MarkerArray` | Tracks of reflectors.                           |
+| `text_markers`                | `visualization_msgs::msg::Marker`      | Text markers that show the calibration metrics. |
+| `calibration_metrics`         | `std_msgs::msg::Float32MultiArray`     | Calibration metrics.                            |
+
+### Services
+
+| Name                       | Type                                                  | Description                                                                              |
+| -------------------------- | ----------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `extrinsic_calibration`    | `tier4_calibration_msgs::` `srv::ExtrinsicCalibrator` | Generic calibration service. The call is blocked until the calibration process finishes. |
+| `extract_background_model` | `std_srvs::srv::Empty`                                | Start to extract the background model from radar and lidar data.                         |
+| `add_lidar_radar_pair`     | `std_srvs::srv::Empty`                                | Add lidar-radar pairs for calibration.                                                   |
+| `delete_lidar_radar_pair`  | `std_srvs::srv::Empty`                                | Delete the previous lidar-radar pair.                                                    |
+| `send_calibration`         | `std_srvs::srv::Empty`                                | Send the calibration result to the sensor calibration manager.                           |
+
+## Parameters
+
+### Core Parameters
+
+| Name                                        | Type          | Default Value                                           | Description                                                                                                                                                        |
+| ------------------------------------------- | ------------- | ------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `radar_parallel_frame`                      | `std::string` | `base_link`                                             | The frame that the radar frame optimizes the transformation to.                                                                                                    |
```

Comment:
the frame that the radar frame sounds redundant. Please rewrite it in a clearer way

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/2eb4dfb32432312ac5cf96ab3a87b5adac1e49fd